app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+class InventoriesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @current_user = current_user
+    @inventories = Inventory.where(user_id: @current_user.id)
+  end
+
+  def show
```

Comment:
- [ ] Kindly fix the inventory details page when I go to it I get this error.
![image](https://github.com/lorenzoworx/chef-chapters/assets/24703171/229879d5-fd0c-442c-b0b3-dbcfe4f28ae7)